model/pubsub.go

```diff
@@ -0,0 +1,9 @@
+package model
+
+// PubsubMsg is the message recevied by the subscriber
+type PubsubMsg struct {
+	Data    []byte
+	Subject string
+}
+
+type PubsubCallback func(subject string, msg *PubsubMsg) ()
```

Comment:
Do we need to pass the subject here when its already present in the struct? I guess we should remove it from the struct